modules/auxiliary/scanner/http/vicidial_sql_enum_users_pass.rb

```diff
@@ -0,0 +1,101 @@
+##
+# This module requires Metasploit: https://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+class MetasploitModule < Msf::Auxiliary
+  include Msf::Exploit::Remote::HttpClient
+  include Msf::Exploit::SQLi
+
+  def initialize(info = {})
+    super(
+      update_info(
+        info,
+        'Name' => 'Vicidial SQL Injection Time-based Admin Credentials Enumeration',
+        'Description' => %q{
+          This module exploits a time-based SQL injection vulnerability in VICIdial, allowing attackers
+          to dump admin credentials (usernames and passwords) via SQL injection.
+        },
+        'Author' => [
+          'Valentin Lobstein', # Metasploit Module
+          'Jaggar Henry of KoreLogic, Inc.' # Vulnerability Discovery
+        ],
+        'License' => MSF_LICENSE,
+        'References' => [
+          ['URL', 'https://korelogic.com/Resources/Advisories/KL-001-2024-011.txt'],
+          ['CVE', '2024-8503']
+        ],
+        'DisclosureDate' => '2024-09-10',
+        'Notes' => {
+          'Stability' => [CRASH_SAFE],
+          'SideEffects' => [IOC_IN_LOGS],
+          'Reliability' => []
+        }
+      )
+    )
+
+    register_options(
+      [
+        Opt::RHOST(),
+        Opt::RPORT(80),
+        OptString.new('TARGETURI', [true, 'Base path of the VICIdial instance', '/']),
+        OptInt.new('SqliDelay', [true, 'Delay in seconds for SQL Injection sleep', 1]),
```

Comment:
Could we use the module metadata to specify this value (and RPORT if necessary)? Currently, this changes the option from advanced to a normal option.
For example (taken from: https://github.com/rapid7/metasploit-framework/blob/1a1c21a0b1685b3f09eec2fe5b808d75a895307a/modules/auxiliary/cloud/aws/enum_ssm.rb#L39C9-L39C58):
```ruby
'DefaultOptions' => { 'SqliDelay' => 1 },
```

Reply:
https://github.com/rapid7/metasploit-framework/pull/19453/commits/04711c44163db939a7782fc5aa09c2d032873ed9